packages/contracts-bedrock/deploy-config/devnetL1.ts

```diff
@@ -2,58 +2,46 @@ import { ethers } from 'ethers'
 
 const { env } = process
 
-const l1GenesisTimestamp =
+const l1GenesisBlockTimestamp =
```

Comment:
We should be able to get rid of this and then parse the value from a flag passed to the hardhat task, we could use the `Math.floor(Date.now() / 1000)` as the default value in the task.

Something like:
if flag passed, use that value, otherwise use the value from the config

Reply:
done